contrib/cirrus/ts.awk

```diff
@@ -0,0 +1,14 @@
+BEGIN {
+    STARTTIME=systime()
+    printf "[%s] START\n", strftime("%T")
+}
+
+{
+    # 12 character prefix aligns $0 on all lines
+    printf "[%+07ds] %s\n", systime()-STARTTIME, $0
```

Comment:
How important is it to align with the start and end times above? Especially if you're displaying only seconds? What do you think of `[%04ds]` instead? It's shorter, hence takes up less space for those reading on a terminal; and it removes the `+`, which I still find unnecessarily confusing (it implies a delta but it's not clear from what).

Reply:
Yes, the extra zeros aren't important, and I wrestled with this myself.  I like the '+' though, since it is a delta counting from `START`.  Hmmm...I have an idea how to make that more apparent.  Let's try this...